pennylane/ops/functions/matrix.py

```diff
@@ -230,6 +230,7 @@ def circuit():
             f"Wires in circuit {list(op.wires)} are inconsistent with "
             f"those in wire_order {list(wire_order)}"
         )
+    QueuingManager.remove(op)
```

Comment:
So `qml.matrix` has been queuing one more `op` all the time?

Reply:
Yep